.github/workflows/build-and-test-orchard-core.yml

```diff
@@ -213,6 +213,10 @@ jobs:
           binary-log-artifact-retention-days: ${{ inputs.build-binary-log-artifact-retention-days }}
           warnings-as-errors: ${{ inputs.warnings-as-errors }}
 
+      - name: Print configuration summary
+        if: always()
+        uses: Lombiq/GitHub-Actions/.github/actions/print-config-summary@issue/OSOE-593
```

Comment:
I would suggest making this step optional with the default `false` and enabled in the super project workflow, since possible this is used by other projects.

Reply:
Let's not overdo this right now. There's no harm in showing that info - and actually I would like it to be enabled by default so that I can access the version info wherever the workflow is run.

We can make it configurable to disable later, but right now, let's please merge it.

Reply:
Noted this under the [issue](https://lombiq.atlassian.net/browse/OSOE-593).